### DISCUSSION TITLE
error deploying monitoring stack

### DISCUSSION THREAD

**Discussion by hitechmuscle** [carlosedp/cluster-monitoring#180] [General]
I'm building a new cluster and installing clustering monitoring on the new cluster.

The ask is, to review what I'm doing to ensure I'm not doing something stupid.

I cloned the project: git clone https://github.com/carlosedp/cluster-monitoring.git and modified the vars.jsonnet file.

Hardware: Raspberry pi 4
OS: Ubuntu 20.04.5 LTS
K3S version: v1.29.4+k3s1

**Verified compatible golang was installed:**
~/cluster-monitoring# go version
go version go1.18.10 linux/arm64

**executed make vendor**

~/cluster-monitoring# make vendor
rm -rf vendor
/root/go/bin/jb install
GET https://github.com/coreos/kube-prometheus/archive/17989b42aa10b1c6afa07043cb05bcd5ae492284.tar.gz 200
GET https://github.com/coreos/etcd/archive/d8c8f903eee10b8391abaef7758c38b2cd393c55.tar.gz 200
GET https://github.com/coreos/prometheus-operator/archive/e31c69f9b5c6555e0f4a5c1f39d0f03182dd6b41.tar.gz 200
GET https://github.com/ksonnet/ksonnet-lib/archive/0d2f82676817bbf9e4acf6495b2090205f323b9f.tar.gz 200
GET https://github.com/kubernetes-monitoring/kubernetes-mixin/archive/b61c5a34051f8f57284a08fe78ad8a45b430252b.tar.gz 200
GET https://github.com/kubernetes/kube-state-metrics/archive/d667979ed55ad1c4db44d331b51d646f5b903aa7.tar.gz 200
GET https://github.com/brancz/kubernetes-grafana/archive/57b4365eacda291b82e0d55ba7eec573a8198dda.tar.gz 200
GET https://github.com/kubernetes/kube-state-metrics/archive/d667979ed55ad1c4db44d331b51d646f5b903aa7.tar.gz 200
GET https://github.com/prometheus/node_exporter/archive/08ce3c6dd430deb51798826701a395e460620d60.tar.gz 200
GET https://github.com/prometheus/prometheus/archive/74207c04655e1fd93eea0e9a5d2f31b1cbc4d3d0.tar.gz 200
GET https://github.com/grafana/grafonnet-lib/archive/8fb95bd89990e493a8534205ee636bfcb8db67bd.tar.gz 200
GET https://github.com/grafana/jsonnet-libs/archive/881db2241f0c5007c3e831caf34b0c645202b4ab.tar.gz 200

**Executed make**

~/cluster-monitoring# make
rm -rf manifests
./scripts/build.sh main.jsonnet /root/go/bin/jsonnet
using jsonnet from arg
+ set -o pipefail
+ rm -rf manifests
+ mkdir -p manifests/setup
++ find manifests -type f '!' -name '*.yaml'
+ find manifests -type f '!' -name '*.yaml' -delete

**executed make deploy** 
~/cluster-monitoring#  make deploy
echo "Deploying stack setup manifests..."
Deploying stack setup manifests...
kubectl apply -f ./manifests/setup/
error: error reading [./manifests/setup/]: recognized file extensions are [.json .yaml .yml]
make: *** [Makefile:44: deploy] Error 1

The make deploy command failed and I observed ~/cluster-monitoring/manifest/setup is empty.

any assistance and/or advice in resolving this issue would be greatly appreciated.

**Comment by tilobormann** [comment]
There is an issue with the build.sh file. Check out the V 0.40 and use this script file, then it works.

**Reply by hitechmuscle** [reply]
using the build.sh from V 0.40, resolved the [Makefile:44: deploy] Error 1] error.  now I'm having trouble accessing Prometheus, Alertmanager, and Grafana at the URLs.  I will troubleshoot and see if I can resolve it.

**Comment by carlosedp** [comment]
Fixed on a8c6bba. Sorry about the delay!
